client/src/modules/templates/bhAccountReferenceSelect.tmpl.html

```diff
@@ -0,0 +1,34 @@
+<div ng-form="{{ $ctrl.name }}" bh-account-reference-select ng-model-options="{ updateOn: 'default' }">
+  <div
+    class="form-group"
+    ng-class="{ 'has-error' : $ctrl.validationTrigger && AccountReferenceForm.account_reference_id.$invalid }">
```

Comment:
Is it possible to remove the $ctrl.validationTrigger, by using the form name.

You can perform that by replacing the `ng-form="{{ $ctrl.name }}"` by something like `ng-form="AccountReferenceForm"`.

And replace all `$ctrl.validationTrigger` by `AccountReferenceForm.$submitted`